docs/UserGuide.md

```diff
@@ -142,6 +142,27 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Marking a patient as visited : `mark`
+
+Marks a specific patient in the record system as having been visited.
+The patient should be specified using an INDEX.
```

Comment:
Hmm are we using index or id number ah? I am a bit confused. If using id number maybe here need to change to id number?

Reply:
Let's stick with ID number since we have already updated it as that in the developer guide

Reply:
Alright, I'll make the changes.